src/api/displayEnums.ts

```diff
@@ -19,3 +19,9 @@ export const displayEngagementStatus = displayEnum<EngagementStatus>();
 export const displayRole = displayEnum<Role>();
 export const displayRoles = (roles: Role[]) =>
   roles.map(displayRole).join(', ');
+
+export const PartnershipStatuses: PartnershipAgreementStatus[] = [
+  'AwaitingSignature',
+  'NotAttached',
```

Comment:
It's a workflow, and not attached is the starting point.
```suggestion
  'NotAttached',
  'AwaitingSignature',
```

Reply:
fixed